src/converter/frame.py

```diff
@@ -29,30 +30,28 @@ def post_process_frame(fig_frame: dict, sketch_artboard: Artboard) -> Artboard:
     # adding always a background rectangle is an overhead for the document itself
     if utils.has_rounded_corners(fig_frame):
         utils.log_conversion_warning("ART001", fig_frame)
-        group.create_clip_mask_if_needed(fig_frame, sketch_artboard)
+        group.create_clip_mask_if_needed(fig_frame, sketch_frame)
 
-    if sketch_artboard.rotation != 0:
+    if sketch_frame.rotation != 0:
         utils.log_conversion_warning("ART002", fig_frame)
 
     # The .fig file clips overlays implicitly but .sketch doesn't, so we must add a mask
-    if sketch_artboard.overlaySettings is not None:
-        sketch_artboard.layers.insert(
-            0, rectangle.make_clipping_rect(fig_frame, sketch_artboard.frame)
-        )
-
-    match sketch_artboard.style.fills:
-        case [Fill(fillType=FillType.COLOR, color=color, isEnabled=True)]:
-            # Single color, apply to artboard
-            sketch_artboard.backgroundColor = color
-            sketch_artboard.hasBackgroundColor = True
-            sketch_artboard.style.fills = []
-
-    if sketch_artboard.style.fills or sketch_artboard.style.borders:
-        # Anything else, add a background rect
-        utils.log_conversion_warning("ART003", fig_frame)
-        group.convert_frame_style(fig_frame, sketch_artboard)
-
-    return sketch_artboard
+    if sketch_frame.overlaySettings is not None:
+        sketch_frame.layers.insert(0, rectangle.make_clipping_rect(fig_frame, sketch_frame.frame))
+
+    # match sketch_frame.style.fills:
+    #     case [Fill(fillType=FillType.COLOR, color=color, isEnabled=True)]:
+    #         # Single color, apply to artboard
+    #         sketch_frame.backgroundColor = color
+    #         sketch_frame.hasBackgroundColor = True
+    #         sketch_frame.style.fills = []
+
+    # if sketch_frame.style.fills or sketch_frame.style.borders:
+    #     # Anything else, add a background rect
+    #     utils.log_conversion_warning("ART003", fig_frame)
+    #     group.convert_frame_style(fig_frame, sketch_frame)
```

Comment:
I want to test this a little more before removing it. Essentially, frames have support fills as part of the style, so we don't have to do this special Artboard background stuff anymore.